Install issues with Fedora 25 and Tern

I have been encountering this issue with Fedora 25 and Tern while using the installer script. Tern is the only package that has an issue while installing.

```
npm WARN deprecated jade@0.26.3: Jade has been renamed to pug, please install the latest version of pug instead of jade
npm ERR! git clone --template=/home/mjackson/.npm/_git-remotes/_templates --mirror git://github.com/lennartcl/tern.git /home/mjackson/.npm/_git-remotes/git-github-com-lennartcl-tern-git-97464df789dbb4d81ca457938
3a02b320c69563d-8a653861: Cloning into bare repository '/home/mjackson/.npm/_git-remotes/git-github-com-lennartcl-tern-git-97464df789dbb4d81ca4579383a02b320c69563d-8a653861'...
npm ERR! git clone --template=/home/mjackson/.npm/_git-remotes/_templates --mirror git://github.com/lennartcl/tern.git /home/mjackson/.npm/_git-remotes/git-github-com-lennartcl-tern-git-97464df789dbb4d81ca457938
3a02b320c69563d-8a653861: fatal: unable to connect to github.com:
npm ERR! git clone --template=/home/mjackson/.npm/_git-remotes/_templates --mirror git://github.com/lennartcl/tern.git /home/mjackson/.npm/_git-remotes/git-github-com-lennartcl-tern-git-97464df789dbb4d81ca457938
3a02b320c69563d-8a653861: github.com[0: 192.30.255.112]: errno=Connection timed out
npm ERR! git clone --template=/home/mjackson/.npm/_git-remotes/_templates --mirror git://github.com/lennartcl/tern.git /home/mjackson/.npm/_git-remotes/git-github-com-lennartcl-tern-git-97464df789dbb4d81ca457938
3a02b320c69563d-8a653861: github.com[1: 192.30.255.113]: errno=Connection timed out
npm ERR! Linux 4.10.12-200.fc25.x86_64
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "install"
npm ERR! node v7.10.0
npm ERR! npm  v4.2.0
npm ERR! code 128

npm ERR! Command failed: git clone --template=/home/mjackson/.npm/_git-remotes/_templates --mirror git://github.com/lennartcl/tern.git /home/mjackson/.npm/_git-remotes/git-github-com-lennartcl-tern-git-97464df78
9dbb4d81ca4579383a02b320c69563d-8a653861
npm ERR! Cloning into bare repository '/home/mjackson/.npm/_git-remotes/git-github-com-lennartcl-tern-git-97464df789dbb4d81ca4579383a02b320c69563d-8a653861'...
npm ERR! fatal: unable to connect to github.com:
npm ERR! github.com[0: 192.30.255.112]: errno=Connection timed out
npm ERR! github.com[1: 192.30.255.113]: errno=Connection timed out
npm ERR! 
npm ERR! 
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /home/mjackson/.npm/_logs/2017-05-12T14_31_24_829Z-debug.log
```

```
tern_from_ts
npm ERR! git clone --template=/home/mjackson/.npm/_git-remotes/_templates --mirror git://github.com/lennartcl/tern.git /home/mjackson/.npm/_git-remotes/git-github-com-lennartcl-tern-git-97464df789dbb4d81ca457938
3a02b320c69563d-8a653861: Cloning into bare repository '/home/mjackson/.npm/_git-remotes/git-github-com-lennartcl-tern-git-97464df789dbb4d81ca4579383a02b320c69563d-8a653861'...
npm ERR! git clone --template=/home/mjackson/.npm/_git-remotes/_templates --mirror git://github.com/lennartcl/tern.git /home/mjackson/.npm/_git-remotes/git-github-com-lennartcl-tern-git-97464df789dbb4d81ca457938
3a02b320c69563d-8a653861: fatal: unable to connect to github.com:
npm ERR! git clone --template=/home/mjackson/.npm/_git-remotes/_templates --mirror git://github.com/lennartcl/tern.git /home/mjackson/.npm/_git-remotes/git-github-com-lennartcl-tern-git-97464df789dbb4d81ca457938
3a02b320c69563d-8a653861: github.com[0: 192.30.255.113]: errno=Connection timed out
npm ERR! git clone --template=/home/mjackson/.npm/_git-remotes/_templates --mirror git://github.com/lennartcl/tern.git /home/mjackson/.npm/_git-remotes/git-github-com-lennartcl-tern-git-97464df789dbb4d81ca457938
3a02b320c69563d-8a653861: github.com[1: 192.30.255.112]: errno=Connection timed out
npm ERR! Linux 4.10.12-200.fc25.x86_64
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "install" "--loglevel" "warn" "tern_from_ts"
npm ERR! node v7.10.0
npm ERR! npm  v4.2.0
npm ERR! code 128

npm ERR! Command failed: git clone --template=/home/mjackson/.npm/_git-remotes/_templates --mirror git://github.com/lennartcl/tern.git /home/mjackson/.npm/_git-remotes/git-github-com-lennartcl-tern-git-97464df78
9dbb4d81ca4579383a02b320c69563d-8a653861
npm ERR! Cloning into bare repository '/home/mjackson/.npm/_git-remotes/git-github-com-lennartcl-tern-git-97464df789dbb4d81ca4579383a02b320c69563d-8a653861'...
npm ERR! fatal: unable to connect to github.com:
npm ERR! github.com[0: 192.30.255.113]: errno=Connection timed out
npm ERR! github.com[1: 192.30.255.112]: errno=Connection timed out
npm ERR! 
npm ERR! 
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /home/mjackson/.npm/_logs/2017-05-12T14_36_48_409Z-debug.log

```

```
0 info it worked if it ends with ok
1 verbose cli [ '/usr/bin/node',
1 verbose cli   '/usr/bin/npm',
1 verbose cli   'install',
1 verbose cli   '--loglevel',
1 verbose cli   'warn',
1 verbose cli   'tern_from_ts' ]
2 info using npm@4.2.0
3 info using node@v7.10.0
4 silly loadCurrentTree Starting
5 silly install loadCurrentTree
6 silly install readLocalPackageData
7 silly fetchPackageMetaData tern_from_ts@git+https://github.com/cloud9ide/tern_from_ts.git#84d51dcb9b16b126a206789d4d4237cde2801fe4
8 silly fetchOtherPackageData tern_from_ts@git+https://github.com/cloud9ide/tern_from_ts.git#84d51dcb9b16b126a206789d4d4237cde2801fe4
9 silly cache add args [ 'tern_from_ts@git+https://github.com/cloud9ide/tern_from_ts.git#84d51dcb9b16b126a206789d4d4237cde2801fe4',
9 silly cache add   null ]
10 verbose cache add spec tern_from_ts@git+https://github.com/cloud9ide/tern_from_ts.git#84d51dcb9b16b126a206789d4d4237cde2801fe4
11 silly cache add parsed spec Result {
11 silly cache add   raw: 'tern_from_ts@git+https://github.com/cloud9ide/tern_from_ts.git#84d51dcb9b16b126a206789d4d4237cde2801fe4',
11 silly cache add   scope: null,
11 silly cache add   escapedName: 'tern_from_ts',
11 silly cache add   name: 'tern_from_ts',
11 silly cache add   rawSpec: 'git+https://github.com/cloud9ide/tern_from_ts.git#84d51dcb9b16b126a206789d4d4237cde2801fe4',
11 silly cache add   spec: 'git+https://github.com/cloud9ide/tern_from_ts.git#84d51dcb9b16b126a206789d4d4237cde2801fe4',
11 silly cache add   type: 'hosted',
11 silly cache add   hosted:
11 silly cache add    { type: 'github',
11 silly cache add      ssh: 'git@github.com:cloud9ide/tern_from_ts.git#84d51dcb9b16b126a206789d4d4237cde2801fe4',
11 silly cache add      sshUrl: 'git+ssh://git@github.com/cloud9ide/tern_from_ts.git#84d51dcb9b16b126a206789d4d4237cde2801fe4',
11 silly cache add      httpsUrl: 'git+https://github.com/cloud9ide/tern_from_ts.git#84d51dcb9b16b126a206789d4d4237cde2801fe4',
11 silly cache add      gitUrl: 'git://github.com/cloud9ide/tern_from_ts.git#84d51dcb9b16b126a206789d4d4237cde2801fe4',
11 silly cache add      shortcut: 'github:cloud9ide/tern_from_ts#84d51dcb9b16b126a206789d4d4237cde2801fe4',
11 silly cache add      directUrl: 'https://raw.githubusercontent.com/cloud9ide/tern_from_ts/84d51dcb9b16b126a206789d4d4237cde2801fe4/package.json' } }
12 verbose addRemoteGit caching git+https://github.com/cloud9ide/tern_from_ts.git#84d51dcb9b16b126a206789d4d4237cde2801fe4
13 verbose addRemoteGit git+https://github.com/cloud9ide/tern_from_ts.git#84d51dcb9b16b126a206789d4d4237cde2801fe4 is a repository hosted by github
14 silly tryClone cloning git+https://github.com/cloud9ide/tern_from_ts.git#84d51dcb9b16b126a206789d4d4237cde2801fe4 via git+https://github.com/cloud9ide/tern_from_ts.git#84d51dcb9b16b126a206789d4d4237cde2801fe4
15 verbose tryClone git-https-github-com-cloud9ide-tern-from-ts-git-84d51dcb9b16b126a206789d4d4237cde2801fe4-8817b282 not in flight; caching
16 verbose correctMkdir /home/mjackson/.npm/_git-remotes correctMkdir not in flight; initializing
17 info git [ 'config', '--get', 'remote.origin.url' ]
18 silly validateExistingRemote git+https://github.com/cloud9ide/tern_from_ts.git#84d51dcb9b16b126a206789d4d4237cde2801fe4 remote.origin.url: https://github.com/cloud9ide/tern_from_ts.git
19 verbose validateExistingRemote git+https://github.com/cloud9ide/tern_from_ts.git#84d51dcb9b16b126a206789d4d4237cde2801fe4 is updating existing cached remote /home/mjackson/.npm/_git-remotes/git-https-github-com-cloud9ide-tern-from-ts-git-84d51dcb9b16b126a206789d4d4237cde2801fe4-8817b282
20 info git [ 'fetch', '-a', 'origin' ]
21 verbose updateRemote git fetch -a origin (https://github.com/cloud9ide/tern_from_ts.git)
22 verbose correctMkdir /home/mjackson/.npm/_git-remotes correctMkdir not in flight; initializing
23 verbose setPermissions git+https://github.com/cloud9ide/tern_from_ts.git#84d51dcb9b16b126a206789d4d4237cde2801fe4 set permissions on /home/mjackson/.npm/_git-remotes/git-https-github-com-cloud9ide-tern-from-ts-git-84d51dcb9b16b126a206789d4d4237cde2801fe4-8817b282
```

--- a/package.json
+++ b/package.json
@@ -12,8 +12,5 @@
   "bugs": {
     "url": "https://github.com/cloud9ide/tern_from_ts/issues"
   },
-  "homepage": "https://github.com/cloud9ide/tern_from_ts",
-  "dependencies": {
-    "tern": "git://github.com/lennartcl/tern.git#97464df789dbb4d81ca4579383a02b320c69563d"
-  }
+  "homepage": "https://github.com/cloud9ide/tern_from_ts"
 }

